src/main/java/de/zalando/aruha/nakadi/domain/EventType.java

```diff
@@ -44,6 +42,9 @@
     @NotNull
     private EventTypeSchema schema;
 
+    @Valid
+    private EventTypeStatistics defaultStatistic;
```

Comment:
I think we forgot to tell you:
We have an agreement that if something can be null it should be markes as `@Nullable`
(so everything is none-null by default, and we avoid having @Nonnull everywhere)